components/layout/footer/socials-section.tsx

```diff
@@ -0,0 +1,78 @@
+import { Box, Typography } from '@interest-protocol/ui-kit';
+
+import { DiscordSVG, GithubSVG, TelegramSVG, XSVG } from '@/components/svg';
+const SocialSection = () => (
+  <Box my="2xl">
+    <Box fontFamily="Proto">
+      <Typography
+        size={'medium'}
+        variant={'body'}
+        fontWeight="500"
```

Comment:
If you added the right size and variant you don't need fontWeight and fontFamily